core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -232,7 +235,11 @@ object PongTLV extends TLVFactory[PongTLV] {
 }
 
 sealed trait EventDescriptorTLV extends TLV {
+
   def noncesNeeded: Int
+
+  def outcomes: Vector[String]
```

Comment:
I will add a scaladoc to this, but this is what the oracle signs. Perhaps I should rename to `oracleFormattedOutcoues`? that's a mouthful but it is what this method represents. 

Any better ideas for names?